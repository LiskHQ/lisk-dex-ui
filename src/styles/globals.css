:root {
  font-size: 16px;
}

html,
body,
#__next {
  padding: 0 !important;
  margin: 0;
  height: 100%;
}

a {
  color: inherit;
  text-decoration: none;
}

* {
  box-sizing: border-box;
}

a,
.btn,
button,
span,
p,
i,
input,
select,
textarea,
li,
img,
svg path,
*::after,
*::before,
.transition-3,
h1,
h2,
h3,
h4,
h5,
h6 {
  transition: all 0.3s ease-out 0s;
}

span,
p,
a,
li.MuiMenuItem-root,
label,
label.MuiFormLabel-root,
span.MuiTypography-root,
p.MuiTypography-root,
button.MuiButton-root,
button.MuiButtonBase-root,
.MuiInput-root,
.MuiTableCell-root {
  font-family: 'Roboto', sans-serif;
}

h1,
h2,
h3,
h4,
h5,
h6,
h1.MuiTypography-root,
h2.MuiTypography-root,
h3.MuiTypography-root,
h4.MuiTypography-root,
h5.MuiTypography-root,
h6.MuiTypography-root {
  font-family: 'Roboto', sans-serif;
}