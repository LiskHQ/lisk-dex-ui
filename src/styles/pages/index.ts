import { GovernanceStyle } from "./governance/index.style";

export {
  GovernanceStyle
}