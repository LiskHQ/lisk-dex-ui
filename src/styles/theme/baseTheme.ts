export const baseTheme = {
  typography: {
    h1: {
      fontSize: "2.5rem",
      fontWeight: 600,
    },
    h2: {
      fontSize: "2rem",
      fontWeight: 600,
    },
    h3: {
      fontSize: "1.75rem",
      lineHeight: "2rem",
      fontWeight: 500,
    },
    h4: {
      fontSize: "1.25rem",
      fontWeight: 600,
    },
    h5: {
      fontSize: "1rem",
      lineHeight: "1.25rem",
      fontWeight: 500,
    },
    h6: {
      fontSize: "0.875rem",
      lineHeight: "1rem",
      fontWeight: 500,
    },
    body1: {
      fontSize: "1rem",
      lineHeight: "1.375rem",
      fontWeight: 500,
    },
    body2: {
      fontSize: "0.875rem",
      lineHeight: "1.125rem",
      fontWeight: 400,
    },
    caption: {
      fontSize: "0.875rem",
      lineHeight: "1rem",
      fontWeight: 500,
    },
    subtitle1: {
      fontSize: "1.75rem",
      lineHeight: "2rem",
      fontWeight: 500,
    },
    subtitle2: {
      fontSize: "1.25rem",
      lineHeight: "1.5rem",
      fontWeight: 400,
    }
  },
  icons: {
    sm: "1rem",
    md: "1.25rem",
    lg: "1.5rem",
  },
  breakpoints: {
    values: {
      xs: 0,
      sm: 576,
      md: 768,
      lg: 1024,
      xl: 1440,
    },
  },
  shadows: {
    sm: "0px 1px 3px rgba(0, 0, 0, 0.1)"
  }
}