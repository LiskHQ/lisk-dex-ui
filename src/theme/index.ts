export { default as theme } from "./theme";
export { default as variables } from "./variables";
