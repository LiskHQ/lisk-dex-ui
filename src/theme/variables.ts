const variables = {
  black: "#000000",
  white: "#ffffff",
  primary: {
    superPurple: "#6953F4",
    lighter: "#5845CC",
    light: "#4738A6",
    main: "#362B7F",
    dark: "#261E59",
    superDark: "#161133",
    contrastText: "#fff",
    stone: "#323d47",
    darkSilver: "#757575",
  },
  secondary: {
    superPurple: "#F5981B",
    lighter: "#CC7E16",
    light: "#A66712",
    main: "#7F4F0E",
    dark: "#59370A",
    superDark: "#332006",
  },
  opacities: {
    superPurple: "#F5981B",
    lighter: "#CC7E16",
    light: "#A66712",
    main: "#7F4F0E",
    dark: "#59370A",
    superDark: "#332006",
    tokenBackground: "#F2F5F9",
  },
  success: {
    superPurple: "#E3F2E0",
    lighter: "#8DC881",
    light: "#459A33",
  },
  warning: {
    superPurple: "#F8F3D8",
    lighter: "#CC7E16",
    light: "#C9A81C",
  },
  information: {
    superPurple: "#F5981B",
    lighter: "#CC7E16",
    light: "#A66712",
  },
  error: {
    superPurple: "#F5981B",
    lighter: "#CC7E16",
    light: "#A66712",
  },
  background: "#E1E3E6",
  lineColor: "#e0e2f1",
};

export default variables;
