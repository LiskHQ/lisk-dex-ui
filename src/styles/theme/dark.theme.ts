import { createTheme } from '@mui/material/styles';
import { baseTheme } from './baseTheme';

const themeColors = {
  text: {
    primary: '#F2F2F2',
    secondary: '#F2F2F2',
    paragraph: '#CDCDCD',
    body: '#F2F2F2',
    heading: '#4738A6',
    button: '#FFFFFF',
    placeholder: '#A4ABB7',
  },
  lightcurve: {
    0: '#6953F4',
    10: '#5845CC',
    20: '#4738A6',
    40: '#362B7F',
    60: '#261E59',
    80: '#E6E6E6',
  },
  secondary: {
    0: '#F5981B'
  },
  primary: {
    85: '#7E6CF4',
    70: '#4F39B2',
    60: '#271B5B',
    40: '#2D2166',
    20: '#322573',
    10: '#4F39B2',
    5: '#322573',
    2.5: '#25195E',
    1: '#F4F8F9',
    0: '#FFFFFF',
  },
  opacities: {
    5: '#333333',
    10: '#3D3D3D',
    20: '#E6E6E6',
    40: '#7E7E7E',
    80: '#D4D4D4',
  },
  border: {
    primary: '#433199',
  },
  bg: {
    primary: 'linear-gradient(90deg, #140C42 0%, #211953 52.88%, #110A3A 101.53%, #140C42 101.53%)',
    secondary: '#22184C',
    helper: '#6953F4',
    proposalItem: 'linear-gradient(180deg, #2E2567 0%, rgba(37, 26, 103, 0.5) 143.01%);',
    banner: 'linear-gradient(180deg, #150556 0%, rgba(25, 9, 98, 0.9) 100%)',
    walletAddress: 'linear-gradient(180deg, #372589 0%, rgba(27, 19, 68, 0.8) 100%)',
    slider: '#322573',
  },
  success: {
    primary: '#E3F2E0',
    second: '#8DC881',
  },
  warning: {
    primary: '#F8F3D8',
    second: '#E2CA64',
  },
  info: {
    primary: '#2EA3DD',
    second: '#C0ECE8',
  },
  error: {
    primary: '#F6E4E6',
    second: '#FB8B96',
  },
} as const;

const lightTheme = createTheme({
  ...themeColors,
  ...baseTheme,
  palette: {
    mode: 'dark',
  },
})

export default lightTheme
