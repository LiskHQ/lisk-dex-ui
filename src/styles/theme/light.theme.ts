import { createTheme } from '@mui/material/styles';
import { baseTheme } from './baseTheme';

const themeColors = {
  text: {
    primary: '#282828',
    secondary: '#515661',
    paragraph: '#6B7280',
    body: '#6B7280',
    heading: '#4738A6',
    button: '#FFFFFF',
    placeholder: '#6B7280',
  },
  lightcurve: {
    0: '#6953F4',
    10: '#A196F6',
    20: '#4738A6',
    40: '#362B7F',
    60: '#261E59',
    80: '#161133',
  },
  primary: {
    85: '#7E6CF4',
    60: '#A196F6',
    40: '#E4EBF9',
    20: '#D9D8F8',
    10: '#E7E9F9',
    5: '#EEF0F8',
    2.5: '#F2F5F9',
    1: '#F4F8F9',
    0: '#6953F4'
  },
  opacities: {
    5: '#333333',
    10: '#3D3D3D',
    20: '#535353',
    40: '#7E7E7E',
    80: '#D4D4D4',
  },
  border: {
    primary: '#E6E6E6',
  },
  bg: {
    primary: '#FBFBFB',
    secondary: '#FFFFFF',
    helper: '#22184D',
    banner: 'linear-gradient(180deg, #F3F3FD 0%, rgba(238, 238, 249, 0.9) 100%)',
    walletAddress: 'linear-gradient(180deg, #EEEFF7 0%, rgba(232, 234, 245, 0.8) 100%);',
    slider: '#F2F2F2',
  },
  success: {
    primary: '#E3F2E0',
    second: '#8DC881',
  },
  warning: {
    primary: '#F8F3D8',
    second: '#E2CA64',
  },
  info: {
    primary: '#EBF7F6',
    second: '#C0ECE8',
  },
  error: {
    primary: '#F6E4E6',
    second: '#FB8B96',
  },
} as const;

const lightTheme = createTheme({
  ...themeColors,
  ...baseTheme,
  palette: {
    mode: 'light'
  },
})

export default lightTheme
