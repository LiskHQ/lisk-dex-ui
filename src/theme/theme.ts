import {
  createTheme,
  responsiveFontSizes,
  ThemeOptions,
} from "@mui/material/styles";

import variables from "./variables";

const options: ThemeOptions = {
  palette: {
    primary: variables.primary,
    secondary: variables.secondary,
  },
  typography: {
    h1: {
      fontSize: "2.5rem",
      fontWeight: 600,
    },
    h2: {
      fontSize: "2rem",
      fontWeight: 400,
    },
    h3: {
      fontSize: "1.75rem",
      fontWeight: 400,
    },
    h4: {
      fontSize: "1.25rem",
      fontWeight: 400,
    },
    h5: {
      fontSize: "1rem",
      fontWeight: 400,
    },
    h6: {
      fontSize: "0.75rem",
      fontWeight: 400,
    },
  },
  breakpoints: {
    values: {
      xs: 0,
      sm: 576,
      md: 768,
      lg: 1024,
      xl: 1440,
      xxl: 1950,
      xxxl: 2300,
    },
  },
};

const theme = responsiveFontSizes(createTheme(options));
export default theme;
