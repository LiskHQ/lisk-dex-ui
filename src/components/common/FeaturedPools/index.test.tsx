import { ThemeProvider } from '@mui/material';
import { fireEvent, render } from '@testing-library/react';
import { IFeaturedPoolsProps, FeaturedPools } from './index';
import React from 'react';
import { lightTheme } from 'styles/theme';

function renderComponent(props: IFeaturedPoolsProps) {
  return render(
    <ThemeProvider theme={lightTheme}>
      <FeaturedPools {...props} />
    </ThemeProvider>
  );
}

describe('FeaturedPools component', () => {
  const mockProps: IFeaturedPoolsProps = {
    onSelectPool: jest.fn(),
  };
  it('checks if the component matches the snapshot', () => {
    const { container } = renderComponent(mockProps);
    expect(container).toMatchSnapshot();
  });

  it('check click feature pool item work', () => {
    const { getByTestId } = renderComponent(mockProps);
    fireEvent.click(getByTestId('feature-pool-item-0'));

    expect(mockProps.onSelectPool).toBeCalledWith('0');
  });
});

