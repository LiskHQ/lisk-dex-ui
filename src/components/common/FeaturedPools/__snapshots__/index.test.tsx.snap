// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeaturedPools component checks if the component matches the snapshot 1`] = `
.emotion-0 {
  position: relative;
}

.emotion-0 .feature-pools-container {
  overflow: auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  scroll-behavior: smooth;
  -webkit-transition: all 1s ease-in-out;
  transition: all 1s ease-in-out;
}

.emotion-0 .feature-pools-container::-webkit-scrollbar {
  display: none;
}

.emotion-0 .scroll-button {
  position: absolute;
  top: 50%;
  -webkit-transform: translateY(-50%);
  -moz-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  width: 2.5rem;
  height: 2.5rem;
}

.emotion-0 .scroll-button.left {
  left: -2.5rem;
}

@media (max-width:767.95px) {
  .emotion-0 .scroll-button.left {
    left: 0rem;
  }
}

.emotion-0 .scroll-button.right {
  right: -2.5rem;
}

@media (max-width:767.95px) {
  .emotion-0 .scroll-button.right {
    right: 0rem;
  }
}

.emotion-0 .scroll-button svg {
  color: #6953F4;
}

.emotion-0 .pool-box {
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  padding: 2px;
  border-radius: 0.5rem;
  background: linear-gradient(90deg, rgba(66, 112, 245, 0.8) 14.06%, rgba(83, 120, 228, 0.8) 36.98%, rgba(231, 36, 176, 0.9) 81.25%);
  margin-right: 2.5rem;
}

.emotion-0 .pool-box:hover {
  background: #0C152E;
}

.emotion-0 .pool-box:hover .pool-box-inner {
  background: #261E59;
}

.emotion-0 .pool-box:hover .token-name {
  color: #FFFFFF;
}

.emotion-0 .pool-box:hover .pool-detail-value {
  color: #FFFFFF;
}

.emotion-0 .pool-box .pool-box-inner {
  width: 18rem;
  height: 10.5rem;
  box-sizing: border-box;
  padding: 1.5rem;
  border-radius: 0.5rem;
  background: #FFFFFF;
}

.emotion-0 .pool-box .pool-tokens {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-0 .pool-box .pool-tokens .token2-image {
  margin-left: -0.75rem;
}

.emotion-0 .pool-box .pool-tokens .token-name {
  margin-left: 1.25rem;
}

.emotion-0 .pool-box .pool-tokens .token-name h4 {
  font-weight: 500;
}

.emotion-0 .pool-box .pool-detail {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  margin-top: 2rem;
}

.emotion-0 .pool-box .pool-detail .pool-detail-title {
  color: #6B7280;
}

.emotion-1 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  text-align: center;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  font-size: 1.5rem;
  padding: 8px;
  border-radius: 50%;
  overflow: visible;
  color: rgba(0, 0, 0, 0.54);
  -webkit-transition: background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
}

.emotion-1::-moz-focus-inner {
  border-style: none;
}

.emotion-1.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-1 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-1:hover {
  background-color: rgba(0, 0, 0, 0.04);
}

@media (hover: none) {
  .emotion-1:hover {
    background-color: transparent;
  }
}

.emotion-1.Mui-disabled {
  background-color: transparent;
  color: rgba(0, 0, 0, 0.26);
}

.emotion-2 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

.emotion-10 {
  margin: 0;
  font-size: 1.25rem;
  line-height: 1.5rem;
  font-weight: 600;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0.00735em;
}

.emotion-14 {
  margin: 0;
  font-size: 0.875rem;
  line-height: 1.125rem;
  font-weight: 400;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  letter-spacing: 0.01071em;
}

<div>
  <div
    class="featured-pools emotion-0"
  >
    <button
      class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium scroll-button left emotion-1"
      tabindex="0"
      type="button"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-chevron-left "
        data-icon="chevron-left"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 320 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 233.4c-12.5 12.5-12.5 32.8 0 45.3l192 192c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L77.3 256 246.6 86.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0l-192 192z"
          fill="currentColor"
        />
      </svg>
      <span
        class="MuiTouchRipple-root emotion-2"
      />
    </button>
    <div
      class="feature-pools-container MuiBox-root emotion-3"
    >
      <div
        class="pool-box MuiBox-root emotion-3"
        data-testid="feature-pool-item-0"
      >
        <div
          class="pool-box-inner MuiBox-root emotion-3"
        >
          <div
            class="pool-tokens MuiBox-root emotion-3"
          >
            <div
              class="token1-image MuiBox-root emotion-3"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2748%27%20height=%2748%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <div
              class="token2-image MuiBox-root emotion-3"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2748%27%20height=%2748%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <div
              class="token-name MuiBox-root emotion-3"
            >
              <h4
                class="MuiTypography-root MuiTypography-h4 emotion-10"
              >
                LSK
              </h4>
              <h4
                class="MuiTypography-root MuiTypography-h4 emotion-10"
              >
                DEX
              </h4>
            </div>
          </div>
          <div
            class="pool-detail MuiBox-root emotion-3"
          >
            <div
              class="pool-apy MuiBox-root emotion-3"
            >
              <p
                class="MuiTypography-root MuiTypography-body2 pool-detail-title emotion-14"
              >
                APY
              </p>
              <p
                class="MuiTypography-root MuiTypography-body2 pool-detail-value emotion-14"
              >
                3.24
                %
              </p>
            </div>
            <div
              class="pool-liquidity MuiBox-root emotion-3"
            >
              <p
                class="MuiTypography-root MuiTypography-body2 pool-detail-title emotion-14"
              >
                Pool Liquidity
              </p>
              <p
                class="MuiTypography-root MuiTypography-body2 pool-detail-value emotion-14"
              >
                $
                6.06
              </p>
            </div>
            <div
              class="pool-apy MuiBox-root emotion-3"
            >
              <p
                class="MuiTypography-root MuiTypography-body2 pool-detail-title emotion-14"
              >
                Fees (24H)
              </p>
              <p
                class="MuiTypography-root MuiTypography-body2 pool-detail-value emotion-14"
              >
                1.5
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="pool-box MuiBox-root emotion-3"
        data-testid="feature-pool-item-1"
      >
        <div
          class="pool-box-inner MuiBox-root emotion-3"
        >
          <div
            class="pool-tokens MuiBox-root emotion-3"
          >
            <div
              class="token1-image MuiBox-root emotion-3"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2748%27%20height=%2748%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <div
              class="token2-image MuiBox-root emotion-3"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2748%27%20height=%2748%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </div>
            <div
              class="token-name MuiBox-root emotion-3"
            >
              <h4
                class="MuiTypography-root MuiTypography-h4 emotion-10"
              >
                LSK
              </h4>
              <h4
                class="MuiTypography-root MuiTypography-h4 emotion-10"
              >
                DEX
              </h4>
            </div>
          </div>
          <div
            class="pool-detail MuiBox-root emotion-3"
          >
            <div
              class="pool-apy MuiBox-root emotion-3"
            >
              <p
                class="MuiTypography-root MuiTypography-body2 pool-detail-title emotion-14"
              >
                APY
              </p>
              <p
                class="MuiTypography-root MuiTypography-body2 pool-detail-value emotion-14"
              >
                11.24
                %
              </p>
            </div>
            <div
              class="pool-liquidity MuiBox-root emotion-3"
            >
              <p
                class="MuiTypography-root MuiTypography-body2 pool-detail-title emotion-14"
              >
                Pool Liquidity
              </p>
              <p
                class="MuiTypography-root MuiTypography-body2 pool-detail-value emotion-14"
              >
                $
                2.3
              </p>
            </div>
            <div
              class="pool-apy MuiBox-root emotion-3"
            >
              <p
                class="MuiTypography-root MuiTypography-body2 pool-detail-title emotion-14"
              >
                Fees (24H)
              </p>
              <p
                class="MuiTypography-root MuiTypography-body2 pool-detail-value emotion-14"
              >
                2.9
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <button
      class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium scroll-button right emotion-1"
      tabindex="0"
      type="button"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-chevron-right "
        data-icon="chevron-right"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 320 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M310.6 233.4c12.5 12.5 12.5 32.8 0 45.3l-192 192c-12.5 12.5-32.8 12.5-45.3 0s-12.5-32.8 0-45.3L242.7 256 73.4 86.6c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0l192 192z"
          fill="currentColor"
        />
      </svg>
      <span
        class="MuiTouchRipple-root emotion-2"
      />
    </button>
  </div>
</div>
`;
